src/components/layouts/ConfirmDeleteUI.tsx

```diff
@@ -14,12 +14,14 @@ interface Prop {
 
 export const ConfirmDeleteUI = ({ handleCancel, handleDelete, body = 'task' }: Prop) => {
   return (
-    <UIContainer>
+    <UIContainer sx={{ width: { xs: 'auto', sm: '470px' } }}>
       <StyledBox>
         <AppMargin size={SizeofAppMargin.MEDIUM} py="20px">
           <Stack direction="column" rowGap={4}>
-            <Typography variant="2xl">Are you sure you want to delete this {body}?</Typography>
-            <Typography variant="lg" sx={{ color: (theme) => theme.color.gray[500] }}>
+            <Typography variant="2xl" sx={{ fontSize: { xs: '18px', sm: '24px' } }}>
+              Are you sure you want to delete this {body}?
```

Comment:
We are changing variant 2xl to use 18 and 25px font sizes that's why the line height is too big. Can we change the variant as well?
![Screenshot_2024-07-12-16-49-41-07_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/8eef42dd-c26b-4f91-9c96-c68a747f8833)

Line height of heading is very large on the above SS.